Updates for several July 2023 mc releases (#964)
minio/docs: source/reference/minio-mc-admin/mc-admin-user-info.rst

- Replace :mc-cmd:`USERNAME <mc admin user info USERNAME>` with the username of the user to display information for.

> The output resembles the following when using the built-in MinIO Identity Provider:

Reviewer: Maybe? (To match the `For ...` below)

```suggestion
For the MinIO internal IDentity Provider (IDP), the output resembles the following:
```

Could link, even. To [MinIO Identity Management](https://min.io/docs/minio/linux/administration/identity-access-management/minio-identity-management.html) unless there's something better?

.. code-block:: shell

   AccessKey: miniouser
   Status: enabled
   PolicyName: 
   MemberOf: []
   Authentication: builtin (miniouser)

> For an third-party LDAP service integration integration, the output resembles the following:

Reviewer: ```suggestion
For a third-party LDAP service integration integration, the output resembles the following:
```
